### ISSUE TITLE
Add a check that looks for concepts used later in DQD

### ISSUE THREAD

**Issue by clairblacketer** [OHDSI/DataQualityDashboard#521]
The DQD makes use of a subset of concepts, like gender, measurement, etc. We should have a check up front that looks to make sure these concepts are in the concept table. If not, the DQD should stop and alert the user.

**Comment by katy-sadowski** [comment]
Good idea, this relates in part to the vocab versioning issues we were discussing last year.  However, perhaps instead of stopping DQD, if we don't find a concept in the concept table the relevant check could be marked NOT_APPLICABLE?
